src/index.ts

```diff
@@ -9,26 +9,29 @@ import './assets/daily.svg';
 import './assets/favicon.ico';
 import './assets/github.png';
 import './assets/new-tab-icon.png';
+import { createRoom } from './create';
 
 function getContainer(): HTMLDivElement {
   return <HTMLDivElement>document.getElementById('container');
 }
 
-function updateJoinedElement(isVisible: boolean) {
+function updateJoinedElement(roomURL: string = null) {
```

Comment:
Is this valid typescript? I'd expect this line to be
```typescript
function updateJoinedElement(roomURL: string | null = null) {
```
Can you double-check with `tsc`?

We could also change it to
```typescript
function updateJoinedElement(roomURL: string = "") {
```
Since `""` is false-y, but that's a pure style choice, I'm indifferent either way.

Reply:
The works because `strictNullChecks` is currently not enabled in compiler options. But now that you point it out, we should opt for strict checking anyway, so I'll enable that and set this to an empty string as you suggest. Good catch!